app/src/main/java/com/willowtree/vocable/MainActivity.kt

```diff
@@ -31,14 +31,12 @@ class MainActivity : BaseActivity() {
         supportActionBar?.hide()
         VocableTextToSpeech.initialize(this)
 
-        if (supportFragmentManager.findFragmentById(R.id.fragment_container) == null) {
-            supportFragmentManager
-                .beginTransaction()
-                .replace(R.id.fragment_container, PresetsFragment())
-                .commit()
-        }
+        val navHost = supportFragmentManager.findFragmentById(R.id.main_nav_host_fragment) as NavHostFragment
+        val navController = navHost.navController
+
+        navController.setGraph(R.navigation.main_nav_graph, intent.extras)
```

Comment:
I don't think you need to call `setGraph` here since you're already setting the `navGraph` property in the activity's xml.